api/modules/automations.js

```diff
@@ -122,50 +60,10 @@ const automations = module.exports = (() => {
         return organizations
     }
 
-    const updateDatePaidPayment = async ({ paymentInformation }) => {
-        const paymentToUpdate = {}
-        if (paymentInformation.external_uuid) {
-            Object.assign(paymentToUpdate, await getPaymentWithExternalId({ id: paymentInformation.external_uuid }))
-        } else {
-            Object.assign(paymentToUpdate, await getPaymentWithId({ id: paymentInformation.id }))
-        }
-
-        paymentToUpdate.date_paid = paymentInformation.date_paid
-        await db.models.Payment.update({
-            date_paid: moment(paymentToUpdate.date_paid, 'YYYY-MM-DD')
-        }, {
-            where: {
-                id: paymentToUpdate.id
-            }
-        })
-
-    }
-
-    const updatePaymentFromStripe = async (params) => {
-        const paymentToUpdate = await db.models.Payment.findOne({
-            where: {
-                external_uuid: params.paymentInformation.external_uuid,
-                external_uuid_type: params.paymentInformation.external_uuid_type
-            }
-        })
-        if (paymentToUpdate) {
-            if (params.paymentInformation.date_paid) {
-                updateDatePaidPayment({ paymentInformation: params.paymentInformation })
-            }
-        } else {
-            //the payment is not in the db, proceed to store it
-            createPayment({ paymentInformation: params.paymentInformation })
-        }
-    }
-
     return {
-        createPayment,
-        deleteDraftInvoicesFromStripe,
         getClientWithExternalId,
```

Comment:
@mecoccaro you should either delete this line or bring back the `getClientWithExternalId` to the file because currently, you are exporting a function that doesn't exist and is causing the server to fail on mount